helper/fileurl_test.go

```diff
@@ -0,0 +1,38 @@
+package helper
```

Comment:
we have the x package for project wide helpers :wink: 
```suggestion
package x
```

Reply:
I have moved this file to the x package